### PR TITLE
Flight fixes

### PR DESCRIPTION
**Pull request by AnastasiaBerlina**

### CODE REVIEW

File: src/main/java/com/epam/lowcost/controller/FlightController.java [hunk 1]
```diff
@@ -79,41 +92,60 @@ public String addNewFlight(Model model) {
...
+        model.addAttribute("flights", flightService.getAllFlightsWithUpdatedPrice());
         model.addAttribute("airports", airportService.getAllAirports());
+        model.addAttribute("currentTime", LocalDateTime.now());
         return SEARCHPAGE;
     }
 
 
-    @RequestMapping(value = SEARCH, method = RequestMethod.GET)
-    public String findFlightByFromToDate(@RequestParam Map<String, String> params, Model model) {
+    public void findFlightByFromToDate(@RequestParam Map<String, String> params, Model model, boolean isAdmin) {
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/controller/FlightController.java]: what for this method ?

File: src/main/resources/messages.properties [hunk 2]
```diff
@@ -85,3 +85,14 @@ lang.updatePlane=Modify plane
 lang.unblockUser=Unblock
 lang.ticketSuccessfullyReturned=Ticket was successfully returned.
 lang.allTicketsDelete=All tickets were successfully returned for flight: 
+lang.allTickets=All bought tickets for flight
+lang.complete=Completed
+lang.findAirportByCity=Find all airports in the city
+NotEmpty=This field is required.
+Size.userForm.username=Please use between 6 and 32 characters.
```
Comment (vakhonin) [inline src/main/resources/messages.properties]: why without lang ?

File: src/main/java/com/epam/lowcost/controller/FlightController.java (hunk 1 above)
Comment (AnastasiaBerlina) [reply src/main/java/com/epam/lowcost/controller/FlightController.java]: if fills model with specific flights , for user with updated prices, for admin with initial ones

File: src/main/resources/messages.properties (hunk 2 above)
Comment (AnastasiaBerlina) [reply src/main/resources/messages.properties]: this constants are for validator

File: src/main/java/com/epam/lowcost/controller/FlightController.java (hunk 1 above)
Comment (vakhonin) [reply src/main/java/com/epam/lowcost/controller/FlightController.java]: I didn't keep in mind mean of this method, I wanted to say about absent of RequestMapping. If it is auxiliary method you can add `private`

File: src/main/resources/messages.properties (hunk 2 above)
Comment (vakhonin) [reply src/main/resources/messages.properties]: And ? Do they have rus version ?

File: src/main/java/com/epam/lowcost/controller/FlightController.java (hunk 1 above)
Comment (AnastasiaBerlina) [reply src/main/java/com/epam/lowcost/controller/FlightController.java]: changed

File: src/main/resources/messages.properties (hunk 2 above)
Comment (AnastasiaBerlina) [reply src/main/resources/messages.properties]: not yet, i dont know whether they should have 'lang.' in front

Comment (AnastasiaBerlina) [reply src/main/resources/messages.properties]: work withtout "lang."

**Pull request merged**